modules/imgcodecs/src/grfmt_tiff.cpp

```diff
@@ -884,19 +884,28 @@ bool TiffEncoder::writeLibTiff( const std::vector<Mat>& img_vec, const std::vect
         int page_compression = compression;
 
         int bitsPerChannel = -1;
+        uint16 sample_format = SAMPLEFORMAT_INT;
         switch (depth)
         {
             case CV_8U:
+                sample_format = SAMPLEFORMAT_UINT;
+            case CV_8S:
```

Comment:
add line with `/* FALLTHRU */` comment before non-break "case".

Here and below.

Details:
- https://stackoverflow.com/questions/45129741/gcc-7-wimplicit-fallthrough-warnings-and-portable-way-to-clear-them
- https://developers.redhat.com/blog/2017/03/10/wimplicit-fallthrough-in-gcc-7

Reply:
ok